content/enterprise/system-overview/capacity.mdx

```diff
@@ -61,6 +61,8 @@ monopolizing the CPU resources of the host.
 The amount of disk storage available to a system plays a small role in the capacity of an instance.
 A root volume with 200 GB of storage can sustain a capacity well over 100 concurrent runs.
 
+
+
```

Comment:
The extra space between the text and the heading isn't needed. It'll render the same as long as there's at least 1 empty line in between.

```suggestion
```